hnn_core/neuron.py

```diff
@@ -139,9 +139,9 @@ def load_custom_mechanisms():
         return
 
     if platform.system() == 'Windows':
-        mech_fname = op.join(op.dirname(__file__), '..', 'mod', 'nrnmech.dll')
+        mech_fname = op.join(op.dirname(__file__), 'mod', 'nrnmech.dll')
```

Comment:
Since this is `__file__` and not `hnn_core.__file__` (as used elsewhere), these changes will fail. Good idea about throwing an exception in this function. Should it check if each mechanism (ar, ca, cad, cat, dipole, dipole_pp, hh2, kca, km) is loaded? PR #108 might have a good way to check that `vecevent.mod` was loaded